### ISSUE TITLE
fasta2fasta.py command-line test fails

### ISSUE THREAD

**Issue by sebastian-luna-valero** [CGATOxford/cgat#99]
Hi all,

This is the error:
```
File "/home/travis/build/CGATOxford/cgat/tests/test_commandline.py", line 137, in check_option
    (script_name, option, map_option2action[option]))
AssertionError: option /home/travis/build/CGATOxford/cgat/scripts/fasta2fasta.py:filter-max-length wrong: action="bad"
```

Log here:
https://s3.amazonaws.com/archive.travis-ci.org/jobs/47254114/log.txt

Could the responsible for this script please have a look?

Best regards,
Sebastian.

**Comment by AndreasHeger** [comment]
I am working on this.

Best wishes,
Andreas

**Comment by AndreasHeger** [comment]
Fixed, 0479a89308a4c766a3f39c5d4f9bf8f5b389746a
